userland/libnrfserialization/Makefile.app

```diff
@@ -18,6 +18,9 @@ endif
 
 HDRS += $(LIBNRFSER_DIR)/headers
 
+$(C_SRCS):	$(HDRS)
```

Comment:
At a certain point you're on the wrong side of the commenting what the code is doing.

That comment reads to me like:
```c
// This code adds two numbers together
int add(int x, int y) {
    return x + y;
}
```

That line of make _literally_ says, the C_SRCS depend on the HDRS - there's a point where comments are just noise